web/styles/dark_theme.css

```diff
@@ -668,6 +680,10 @@
         background-color: hsl(212deg 32% 14%);
     }
 
+    #searchbox_form .dropdown-menu ul {
+        background-color: inherit;
```

Comment:
One thing to watch for is when it's better to genuinely inherit a background color (which gives the element a background color for real) and when it's better to just set it to [`unset`](https://developer.mozilla.org/en-US/docs/Web/CSS/unset)--effectively allowing the background of the containing element to just show through). I don't know what's the better option here, but if `unset` works, it's probably better to stick with it.

Reply:
Cool, new CSS knowledge for me! `unset` does work :)